README.md

```diff
@@ -3033,8 +3033,36 @@ You can then assign a function to a variables like this `x = my_function` or you
 <details>
 <summary>Write a function to determine if a number is a Palindrome</summary><br><b>
 
+- Code:
+
+```
+from typing import Union
+
+def isNumberPalindrome(number: Union[int, str]) -> bool:
+    if isinstance(number, int):
+        number = str(number)
+    return number == number[::-1]
+
+print(isNumberPalindrome("12321"))
+```
+
+- Using Python3.10 that accepts using bitwise operator '|'
+
+```
+def isNumberPalindrome(number: int | str) -> bool:
+    if isinstance(number, int):
+        number = str(number)
+    return number == number[::-1]
```

Comment:
This perhaps the shortest way to do so, but we might want to specify that it's not the fastest

Reply:
To clarify, I meant [::-1], not the bitwise operator